roles/vdm/tasks/assets.yaml

```diff
@@ -79,3 +79,14 @@
     - install
     - uninstall
     - update
+
+- name: assets - Copy user-provided license file
+  copy:
+    src: "{{ V4_CFG_LICENSE}}"
```

Comment:
@enderm is the V4_CFG_LLICENSE location relative to the actual deployment or can I use an absolute path when setting the variable?

Reply:
you can specify absolute paths. I'm actually not so sure about relative one's. Never tried them.